packages/ui/src/globals.css

```diff
@@ -43,7 +35,7 @@ body {
 
 html,
 body {
-  font-family: var(--font-sans);
+  font-family: theme(fontFamily.sans);
```

Comment:
Nice, where is this coming from?

Reply:
It's the `theme()` directive included by Tailwind: https://tailwindcss.com/docs/functions-and-directives#theme